analytics/pyspark/pom.xml

```diff
@@ -42,10 +42,18 @@
                     </execution>
                 </executions>
             </plugin>
+            </plugins>
+        </build>
 
+    <profiles>
+    <profile>
+    <id>python</id>
```

Comment:
@JWileczek I have a PR coming, but I'm pretty sure the profile has to be "pyspark" or we need to change the profile being activated from deploy, right?  Well, thats what I assume and am going to just name this profile "pyspark" so everything should work the same.